apps/templates/pages/users.html

```diff
@@ -255,8 +279,8 @@ <h4 class="modal-title">Confirm Approve Users</h4>
         pageLength: savedLength,
         displayStart: savedPage * savedLength,
 
-        stateSave: true,
-        stateDuration: -1
+        /* stateSave: true,
+        stateDuration: -1 */
```

Comment:
your branch seems outdated. Can you please rebase or merge the changes from branch ? (and possible handle conflicts)

Reply:
fixed